R/financial_functions.R

```diff
@@ -45,23 +45,75 @@ calculate_net_profits <- function(data,
     )
 }
 
+#' Calculates annual net profits on the company-technology level for the
+#' baseline and late and sudden scenarios without carbon tax Climate laggards which need to build
+#' out their production in increasing technologies to compensate for their
```

Comment:
1. please use the roxygen sceletons to add documentation to make sure that they are formatted corretly and complete. Simply click on a function and click Insert Roxygen Sceleton under the Code pain
2. I would suggest to put the details not in calculate_net_profits_without_carbon_tax (here it is enought to say that it calculates the net profits for the following cases) and instead go into detail on the smaller functions.

Reply:
2. makes sense, got it. 1. still needs to be addressed

Reply:
1. done